src/Config_Command.php

```diff
@@ -223,14 +223,35 @@ public function get( $_, $assoc_args ) {
 
 		$wp_cli_original_defined_constants = get_defined_constants();
 		$wp_cli_original_defined_vars      = get_defined_vars();
+		$wp_cli_original_includes          = get_included_files();
 
 		eval( WP_CLI::get_runner()->get_wp_config_code() );
 
 		$wp_config_vars      = self::get_wp_config_vars( get_defined_vars(), $wp_cli_original_defined_vars, 'variable', array( 'wp_cli_original_defined_vars' ) );
 		$wp_config_constants = self::get_wp_config_vars( get_defined_constants(), $wp_cli_original_defined_constants, 'constant' );
 
-		$get_constant        = ! empty( $assoc_args['constant'] );
-		$get_global          = ! empty( $assoc_args['global'] );
+		foreach ( $wp_config_vars as $key => $value ) {
+			if ( 'wp_cli_original_includes' === $value['key'] ) {
+				$key_backup = $key;
+				break;
+			}
+		}
+
+		unset( $wp_config_vars[ $key_backup ] );
+		$wp_config_vars           = array_values( $wp_config_vars );
+		$wp_config_includes       = array_diff( get_included_files(), $wp_cli_original_includes );
+		$wp_config_includes_array = array();
+
+		foreach ( $wp_config_includes as $key => $value ) {
+			$wp_config_includes_array[] = array(
+				'key'   => 'FILE',
```

Comment:
@danielbachhuber The case in the first colum are all CAPS except for table_prefix, so should I leave it as it is?
```
+------------------+------------------------------------------------------------------+----------+
| key              | value                                                            | type     |
+------------------+------------------------------------------------------------------+----------+
| table_prefix     | wp_                                                              | variable |
| DB_NAME          | wpclidemo3                                                       | constant |
| DB_USER          | root                                                             | constant |
| DB_PASSWORD      | AqRcKrLo                                                         | constant |
| DB_HOST          | localhost                                                        | constant |
| DB_CHARSET       | utf8                                                             | constant |
| DB_COLLATE       |                                                                  | constant |
| AUTH_KEY         | <NA1f1=+SO*FHdc?ioBKpHBsby|-z@CDbl?Wi@$eJ*;?&yUpsIJmsTNw0`>X -e! | constant |
| SECURE_AUTH_KEY  | N^vUMl[bI(TWX&&)XS]KQXT_n.))s6,+l,>$5>_vv],]q94[zwx,=x7Xx-=M/Z^f | constant |
| LOGGED_IN_KEY    | 4[3&*,~Nue2%G,kwWC:V~pYE@|{k}{LL6oSit.21pBrI|~b_lP-@j0zqCO|0bv%} | constant |
| NONCE_KEY        | Y9(-tHv^l*W+45+-t)D^hJp$hoM4si4t`k7lABHhrZc&4p%T$8eh_CKh^RB#mo{a | constant |
| AUTH_SALT        | @XO|tzQF+hxDDXcas;F7+9u;A[#JBA}|&Ls|:{||w t:UMj>a6+;g9uV4F2i>NGb | constant |
| SECURE_AUTH_SALT | EA[Y4lmpA{3o!xbs)n-#&JWa7.+Sm)j?Hn}>P+9}(!<Z* 2.GT*yU=)<Bu0W)N+: | constant |
| LOGGED_IN_SALT   | ZB=.?T9mJ4U9?GmiR$tQ0QI&9&O0&vFW-}CaJkLb/v+e11qz7WeBC<o/rm@vVtX` | constant |
| NONCE_SALT       | wa>=Ukt}-&5_ ~6<?JLGb8pjY7`S98~#T0D)#pt&(o:aAi~.EruyC]JhmB.)wRO- | constant |
| FILE             | /home/siddharth/Documents/wpcliwp/sid.php                        | includes |
| FILE             | /home/siddharth/Documents/wpcliwp/prianca.php                    | includes |
+------------------+------------------------------------------------------------------+----------+

```